spec/features/services/create_collection_association_spec.rb

```diff
@@ -0,0 +1,152 @@
+require 'rails_helper'
+
+describe 'Creating Service Collection Associations', js: true, reset_provider: true do
+  before do
+    login
+
+    @ingest_response, _concept_response = publish_service_draft
```

Comment:
This service will be ingested for each test since it's not in the `before :all` block - is that what you wanted?